src/panels/SessionRecordingPanel/PlaySession.tsx

```diff
@@ -75,6 +102,48 @@ export function PlaySession() {
             disabled={!shouldOutputFrames || !isIdle}
           />
         </Group>
+        <BoolInput
+          label={'Hide GUI on playback'}
+          value={hideGuiOnPlayback}
```

Comment:
As already mentioned, this should not be hidden per default. 

Maybe we should also not hide the dashboard per default?